docs/apache-airflow/concepts/deferring.rst

```diff
@@ -109,13 +109,13 @@ There's also some design constraints to be aware of:
 
 * The ``run`` method *must be asynchronous* (using Python's asyncio), and correctly ``await`` whenever it does a blocking operation.
 * ``run`` must ``yield`` its TriggerEvents, not return them. If it returns before yielding at least one event, Airflow will consider this an error and fail any Task Instances waiting on it. If it throws an exception, Airflow will also fail any dependent task instances.
-* A Trigger *must be able to run in parallel* with other copies of itself. This can happen both when two tasks defer based on the same trigger, and also if a network partition happens and Airflow re-launches a trigger on a separated machine.
+* You must assume that duplicates of your trigger instance may run simultaneously.  This can happen if a network partition happens and Airflow re-launches a trigger on a separated machine.
```

Comment:
```suggestion
* You must assume that duplicates of your trigger instance may run simultaneously. This can happen if a network partition happens and Airflow re-launches a trigger on a separated machine.
```

Reply:
It is still not clearer to me tbh -- but I don't have a better suggestion either

Reply:
Which aspect isn't clear? I can try to clarify the wording

Reply:
```
You must assume that your trigger instance may run more than once (this can happen if a network partition happens and Airflow re-launches a trigger on a separated machine).  So you must be mindful about side effects.  E.g. you might not want to use a trigger to insert database rows.
```

Reply:
Yea that is better